helpers/hstring.py

```diff
@@ -94,6 +94,48 @@ def get_docstring_line_indices(lines: List[str]) -> List[int]:
     return docstring_line_indices
 
 
+# TODO(allenmatt10): Add tests.
+def get_function_indices(lines: List[str]) -> List[int]:
+    """
+    Get indices of lines of code that are inside functions and methods.
+
+    :param lines: the code lines to check
+    :return: the indices of function lines
+    """
+    function_line_indices = []
```

Comment:
Let's add some _LOG.debug to help debugging

Our rule is: "if there is a problem, one should enable -v DEBUG and get enough info to debug"